### ISSUE TITLE
Export does not match preview

### ISSUE THREAD

**Issue by ssarl** [icearith/iA-Writer-Templates-Github-Plus#5]
When using export function it does not correctly display any of the additional features (math, syntax highlighting etc.).

Example markdown code: 

```
#include <stdio.h>
int main(){
	/* this is a print statement */
	printf("Hello, World!");
}
```
results to the attached file.
[Test.pdf](https://github.com/icearith/iA-Writer-Templates-Github-Plus/files/1135515/Test.pdf)

**Comment by icearith** [comment]
@ssarl Yeah, I also found this issue recently, I was working on my dissertation recently, and I will look into this issue. Thanks for reporting. Sorry for late reply. :)
